crates/ruff_linter/src/rules/refurb/rules/for_loop_writes.rs

```diff
@@ -0,0 +1,121 @@
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::{Expr, Stmt, StmtFor};
+use ruff_python_semantic::analyze::typing;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks use of `IOBase.write` in a for loop.
+///
+/// ## Why is this bad?
+/// When writing a batch of elements, it's more idiomatic to use a single method call
+/// `IOBase.writelines` rather than write elements one by one.
```

Comment:
```suggestion
/// When writing a batch of elements, it's more idiomatic to use a single method call,
/// `IOBase.writelines`, rather than write elements one by one.
```

Reply:
Great catch, thanks!